CMakeLists.txt

```diff
@@ -42,21 +40,19 @@ set(ASYNCIO_INC
         include/asyncio/finally.h
         )
 
-include_directories(${CMAKE_SOURCE_DIR}/include)
 add_library(asyncio SHARED
         ${ASYNC_INC}
         src/event_loop.cpp)
-target_link_libraries(asyncio PUBLIC fmt::fmt)
 
-add_subdirectory(test)
-add_subdirectory(third_party)
+find_package(fmt REQUIRED)
```

Comment:
github ci can't find the fmt package,  is this line still needed?

Reply:
Ah, I had fmt in my build environment already so I didn't need to check out the submodules.

Having a look at the code, it can be removed as a dependency from the core library quite easily as it's only called in two or three places for trivial things. Do you think this is suitable?

Reply:
I have added libfmt-dev in ci, but it reports error:

```
install(EXPORT "asyncioTargets" ...) includes target "asyncio" which requires target "fmt" that is not in any export set.
```

Reply:
Took a second to figure out what was going on there, but because this is EXPORTing the asyncio target, any dependencies to that target need to be EXPORTed as well (which we can't do for `fmt` as it's a dependency). CMake stuff is always annoying, the simplest fix is to use the system fmt lib or remove the fmt lib dep.